docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -41,9 +41,9 @@ To use Iceberg, you need:
   a {ref}`Nessie server <iceberg-nessie-catalog>`, or a
   {ref}`Snowflake catalog <iceberg-snowflake-catalog>`.
 
-- Data files stored in the file formats {ref}`ORC <hive-orc-configuration>` or
-  {ref}`Parquet <hive-parquet-configuration>` (default) on a [supported file
-  system](iceberg-file-system-configuration).
+- Data files stored in the file formats
+  [Parquet](hive-parquet-configuration)(default), [ORC](hive-orc-configuration),
+  or AVRO on a [supported file system](iceberg-file-system-configuration).
```

Comment:
Rename AVRO to Avro. See https://avro.apache.org/

Reply:
Done.